openmc/data/neutron.py

```diff
@@ -21,6 +22,74 @@
     basestring = str
 
 
+def _get_metadata(zaid, metastable_scheme='nndc'):
+    """Method to obtain the complete element name, element, Z, mass_number,
+    and metastable state
```

Comment:
The first line of a docstring should only be one line long (PEP 257).  Ideally, it should also be formatted like a sentence.  How about "Return basic identifying data for a nuclide with given ZAID."?